src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureColor {
+    private int firstLeg;
+    private int secondLeg;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int height, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg + secondLeg) / 2 * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: " + String.format("%.1f", getArea())
+                + " sq. units, first leg: " + firstLeg + " units, second leg: " + secondLeg
+                + " units, height: " + height + " units, color: "
+                + getColor().toString().toLowerCase());
```

Comment:
Use the 'name()' method instead of 'toString()' when you want to get the String representation of enum constants. This ensures that even if 'toString()' is overridden, the actual name of the enum constant is returned.